<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainAdmin</class>
 <widget class="QMainWindow" name="MainAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu Administration</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Gestion des Plats</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_1_1">
           <attribute name="title">
            <string>Ajouter Plat</string>
           </attribute>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="geometry">
             <rect>
              <x>220</x>
              <y>90</y>
              <width>601</width>
              <height>331</height>
             </rect>
            </property>
            <property name="title">
             <string/>
            </property>
            <widget class="QWidget" name="horizontalLayoutWidget_5">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>40</y>
               <width>461</width>
               <height>80</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Nom du commande</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="horizontalLayoutWidget_6">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>130</y>
               <width>461</width>
               <height>61</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>                     Prix    </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_2"/>
              </item>
             </layout>
            </widget>
            <widget class="QPushButton" name="pushButton">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>210</y>
               <width>221</width>
               <height>91</height>
              </rect>
             </property>
             <property name="text">
              <string>Ajouter image du commande</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>plus.png</normaloff>plus.png</iconset>
             </property>
            </widget>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_1_2">
           <attribute name="title">
            <string>Modifier Plat</string>
           </attribute>
           <widget class="QGroupBox" name="groupBox_7">
            <property name="geometry">
             <rect>
              <x>120</x>
              <y>70</y>
              <width>761</width>
              <height>521</height>
             </rect>
            </property>
            <property name="title">
             <string/>
            </property>
            <widget class="QListView" name="listView">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>50</y>
               <width>701</width>
               <height>281</height>
              </rect>
             </property>
            </widget>
            <widget class="QWidget" name="horizontalLayoutWidget_7">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>350</y>
               <width>461</width>
               <height>61</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Nom du commande</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_3"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="horizontalLayoutWidget_8">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>420</y>
               <width>461</width>
               <height>51</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>                     Prix    </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_4"/>
              </item>
             </layout>
            </widget>
            <widget class="QPushButton" name="pushButton_2">
             <property name="geometry">
              <rect>
               <x>540</x>
               <y>430</y>
               <width>141</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>Modifier</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>settings.png</normaloff>settings.png</iconset>
             </property>
            </widget>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_1_3">
           <attribute name="title">
            <string>Supprimer Plat</string>
           </attribute>
           <widget class="QGroupBox" name="groupBox_8">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>80</y>
              <width>751</width>
              <height>401</height>
             </rect>
            </property>
            <property name="title">
             <string/>
            </property>
            <widget class="QListView" name="listView_2">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>40</y>
               <width>701</width>
               <height>281</height>
              </rect>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_3">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>340</y>
               <width>111</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>Supprimer</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>x-button.png</normaloff>x-button.png</iconset>
             </property>
            </widget>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Gestion des Serveurs</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidget_3">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab_2_1">
           <attribute name="title">
            <string>Ajouter Serveur</string>
           </attribute>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>70</y>
              <width>621</width>
              <height>291</height>
             </rect>
            </property>
            <property name="title">
             <string/>
            </property>
            <widget class="QWidget" name="horizontalLayoutWidget">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>30</y>
               <width>391</width>
               <height>81</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Nom d'utilisateur</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="line1"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="horizontalLayoutWidget_2">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>120</y>
               <width>391</width>
               <height>80</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Mot de passe      </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="line2"/>
              </item>
             </layout>
            </widget>
            <widget class="QPushButton" name="bt1">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>220</y>
               <width>101</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>Ajouter</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../../../../PycharmProjects/coffe/plus.png</normaloff>../../../../PycharmProjects/coffe/plus.png</iconset>
             </property>
            </widget>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_2_2">
           <attribute name="title">
            <string>Modifier Serveur</string>
           </attribute>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>50</y>
              <width>661</width>
              <height>481</height>
             </rect>
            </property>
            <property name="title">
             <string/>
            </property>
            <widget class="QTableWidget" name="table1">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>50</y>
               <width>601</width>
               <height>192</height>
              </rect>
             </property>
            </widget>
            <widget class="QWidget" name="horizontalLayoutWidget_3">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>250</y>
               <width>391</width>
               <height>81</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Nouveau serveur</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="line3"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="horizontalLayoutWidget_4">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>340</y>
               <width>391</width>
               <height>80</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Mot de passe      </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="line4"/>
              </item>
             </layout>
            </widget>
            <widget class="QPushButton" name="bt2">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>440</y>
               <width>101</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>Ajouter</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../../../../PycharmProjects/coffe/plus.png</normaloff>../../../../PycharmProjects/coffe/plus.png</iconset>
             </property>
            </widget>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_2_3">
           <attribute name="title">
            <string>Supprimer Serveur</string>
           </attribute>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>60</y>
              <width>701</width>
              <height>381</height>
             </rect>
            </property>
            <property name="title">
             <string/>
            </property>
            <widget class="QTableView" name="table2">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>50</y>
               <width>631</width>
               <height>261</height>
              </rect>
             </property>
            </widget>
            <widget class="QPushButton" name="bt3">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>330</y>
               <width>111</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>Supprimer</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../../../../PycharmProjects/coffe/x-button.png</normaloff>../../../../PycharmProjects/coffe/x-button.png</iconset>
             </property>
            </widget>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Gestion des tables</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Tables</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Niveaux</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>999</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
